Index: src/main/java/moviemanager/controller/PremiereController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/moviemanager/controller/PremiereController.java b/src/main/java/moviemanager/controller/PremiereController.java
--- a/src/main/java/moviemanager/controller/PremiereController.java	
+++ b/src/main/java/moviemanager/controller/PremiereController.java	(date 1671351888963)
@@ -38,7 +38,8 @@
 
 
     @GetMapping("/list")
-    private String showerList(@PageableDefault(size = 9) Pageable pageable, Model model) {
+    private String showerList(@PageableDefault(size = 9) Pageable pageable,
+                              Model model) {
         model.addAttribute("premierePage", iPremiereService.fildPage(pageable));
         return "/premiere/list";
     }
